handler.go

```diff
@@ -284,10 +270,29 @@ func (fm *Frontman) sendResultsChanToHubWithInterval(resultsChan chan Result) er
 
 func (fm *Frontman) RunOnce(input *Input, outputFile *os.File, interrupt chan struct{}, writeResultsChanToFileContinously bool) error {
 	var err error
+	var hostInfo MeasurementsMap
 
 	// in case HUB server will hang on response we will need a buffer to continue perform checks
 	resultsChan := make(chan Result, 100)
 
+	// since fm.Run calls fm.RunOnce over and over again, we need this check here
```

Comment:
@requilence What do you think about this approach?

Reply:
agree